### DISCUSSION TITLE
Running a recipe

### DISCUSSION THREAD

**Discussion by joeoct93** [asteroid-team/asteroid#625] [Q&A] [answered]
I've been trying to run a recipe using the instructions on the main page.
I put in the following code:
cd egs/wham/ConvTasNet
. ./run.sh --id 0

However, I got the following response, and at the end, the terminal simply shut down. It instantly shut down every time I ran it after that.
Stage 0: Generating Librimix dataset
Cloning into 'LibriMix'...
remote: Enumerating objects: 268, done.
remote: Counting objects: 100% (105/105), done.
remote: Compressing objects: 100% (47/47), done.
remote: Total 268 (delta 68), reused 58 (delta 58), pack-reused 163
Receiving objects: 100% (268/268), 33.42 MiB | 3.85 MiB/s, done.
Resolving deltas: 100% (113/113), done.
Download LibriSpeech/dev-clean into --n_src
Download LibriSpeech/test-clean into --n_src
Download LibriSpeech/train-clean-100 into --n_src
Download LibriSpeech/train-clean-360 into --n_src
Download wham_noise into --n_src

Redirecting output to ‘wget-log’.

Redirecting output to ‘wget-log.1’.

Redirecting output to ‘wget-log.2’.

Redirecting output to ‘wget-log.3’.

Redirecting output to ‘wget-log.4’.
rm: unrecognized option '--n_src/dev-clean.tar.gz'
Try 'rm ./--n_src/dev-clean.tar.gz' to remove the file '--n_src/dev-clean.tar.gz'.
Try 'rm --help' for more information.
[1]   Exit 1                  LibriSpeech_dev_clean
[2]   Exit 4                  LibriSpeech_test_clean
[3]   Exit 4                  LibriSpeech_clean100
[4]-  Exit 4                  LibriSpeech_clean360

I've been stuck in this for over 1 hour. Is this normal? What am I supposed to do?

**Comment by mpariente** [comment, marked as answer]
You need to specify the directory where data will be stored : the `storage_dir` variable at the top of the run.sh script. 

I'm  merging a fix in #626.

**Reply by joeoct93** [reply]
Thank you, this solved my problem
